af04f855-58c4-4db3-a191-45fe33381679/scripts/actions.py

```diff
@@ -799,3 +795,22 @@ def CheckTournamentValid(group, x=0, y=0):
 			notify("{}'s deck is legal in a FFG + Apoka format".format(me))
 	else:
 		notify("{}'s deck does contain some invalid cards for the selected format".format(me))
+
+#------------------------------------------------------------------------------
+# Choose secret Number
+#------------------------------------------------------------------------------
+
+chosenNumber = None
+def askSecretNumber(group, x=0, y=0):
```

Comment:
Are the numbers unbounded and the dial can't be used? I.e. 1-5?

Reply:
How is it played irl? Dice or pen/paper?

Reply:
Pen or d20 dice as long as it's hidden ?